layers/+emacs/org/config.el

```diff
@@ -95,3 +95,6 @@ ATTENTION: `valign-mode' will be laggy working with tables contain more than 100
 
 (defvar org-enable-roam-server nil
   "if non-nil, enable org-roam-server support")
+
+(defvar org-enable-asciidoc-support nil
+  "if non-nil, asciidoc is configured")
```

Comment:
The documentation starts with capital letter and ends with period.

configured -> enabled

Reply:
I was mostly copy pasting from what other people wrote, should I edit the docstrings of some of these other vars?

Reply:
That's very much appreciated!

The not complete checklist:

- Documentation starts with a short sentence describing the functionality, < 70 characters.
- Sentences should start with capital letters and ends with punctuations (usually periods), and don't have trailing whitespace.
- The first paragraph is indented (automatically), but don't indent any following sentences.
- Imperative mood is required by standard. Active voice is better to understand.
- Quote symbol as `SYMBOL', strings as \"STRING\", and don't quote numerals. Don't quote t and nil.

Reply:
> That's very much appreciated!
Ok, I might fix that in a separate commit.

> The not complete checklist:
> 
> * Documentation starts with a short sentence describing the functionality, < 70 characters.
- [x] Yep, I think I've satisfied this.
> * Sentences should start with capital letters and ends with punctuations (usually periods), and don't have trailing whitespace.
- [x] Ok, fixed it.
> * The first paragraph is indented (automatically), but don't indent any following sentences.
- [x] Well, there isn't more than one line to indent.
> * Imperative mood is required by standard. Active voice is better to understand.
- [x] I didn't change "If" to "When" as none of the other entries look like that. Otherwise it's in active voice.
> * Quote symbol as `SYMBOL', strings as "STRING", and don't quote numerals. Don't quote t and nil.
- [x] Ok, I'm not quoting any symbols or strings in the docs, so everything should be alright.